index.js

```diff
@@ -1,5 +1,4 @@
-const { Project, ts } = require("ts-morph");
```

Comment:
If you want this to be super fast, you may consider using the transform api: https://ts-morph.com/manipulation/transforms

Reply:
Good to know, might be important later.